resources/services/codepipeline/pipelines.go

```diff
@@ -12,19 +11,14 @@ import (
 	"github.com/cloudquery/cq-provider-sdk/provider/schema"
 )
 
-type StageWrapper struct {
-	types.StageDeclaration
-	StageOrder int32
-}
-
 //go:generate cq-gen --resource pipelines --config gen.hcl --output .
 func Pipelines() *schema.Table {
 	return &schema.Table{
 		Name:         "aws_codepipeline_pipelines",
-		Description:  "Represents the output of a GetPipeline action.",
+		Description:  "Represents the output of a GetPipeline action",
 		Resolver:     fetchCodepipelinePipelines,
 		Multiplex:    client.ServiceAccountRegionMultiplexer("codepipeline"),
-		IgnoreError:  client.IgnoreCommonErrors,
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
```

Comment:
I think this needs to be updated in the `gen.hcl` file

Reply:
Is `IgnoreCommonErrors` the one we want here?

Reply:
Updated in the gen.hcl and regenerated to use `IgnoreCommonErrors` like it was before.